src/RFC-0122_Burning.md

```diff
@@ -115,6 +115,27 @@ burned\_total = \sum_j\mathrm{C_{j}}  \\\\
 \tag{1}
 $$
 
+### Creation of burned output
+
+When creating the output, we need to be able to generate a Proof of Burn for the burned output. 
+In the header, we already commit to each and every [UTXO] ever created. There already is a proof of the existence
+for this burned output, but this proof is only to the complete output and not just the commitment. 
+
+To get around this, we enforce default values for most of the output except the version and commitment field. 
```

Comment:
I don't like this approach. Feels very hacky

Reply:
I want us to enforce these rules anyway, regardless of the route we go. I don't think there are any security gaps around allowing scripts etc on the burned outputs, but I don't think we should take that chance. As an additional benefit, this is smaller per output, if all the fields are essentially blank.

Reply:
The space doesn't really make a difference, since these are pruned immediately. I'm not convinced these need to be blank

Reply:
Surely the only change from the _status quo_ is that we must commit to the BURN flag somewhere in an MMR so it can never be unburnt. This in addition to a consensus rule forbidding BTXOs to be used as inputs, of course.